test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -221,6 +225,10 @@ void doBlockchainTestNoLog(json_spirit::mValue& _v, bool _fillin)
 		}
 	}
 
+	//Delete source test from the json
+	for (auto i: erase_list)
+		_v.get_obj().erase(i);
```

Comment:
I realized this is actually wrong. `erase()` invalidates map iterators, so only the first erase is ok, then you have the vector of invalid iterators. If the erase list to be used, it must contain map keys, not iterators.